frontend/src/pages/RegisterPage.vue

```diff
@@ -0,0 +1,221 @@
+<template>
+  <wof-card>
+    <div class="register-page">
+      <h1 class="register-page__title">Create Account</h1>
+      <div class="horizontal-line"></div>
+      <form class="register-page__form">
+        <div class="register-page__input">
+          <wof-input name="User name" :error="username.errorMsg" @change="setUsername" :maxLength="20"></wof-input>
+        </div>
+        <div class="register-page__input">
+          <wof-input name="Email address" :error="email.errorMsg" @change="setEmail"></wof-input>
+        </div>
+        <div class="register-page__input">
+          <wof-input name="Password" :type="'password'" :error="password.errorMsg" @change="setPassword"></wof-input>
+        </div>
+        <div class="register-page__input">
+          <wof-input name="Confirm Password" :type="'password'" :error="passwordConfirm.errorMsg" @change="setPasswordConfirm"></wof-input>
```

Comment:
Shall we stick to the lowercase after the first word?
```suggestion
          <wof-input name="Confirm password" :type="'password'" :error="passwordConfirm.errorMsg" @change="setPasswordConfirm"></wof-input>
```